Makefile.am

```diff
@@ -84,7 +84,8 @@ bin_PROGRAMS = \
     tools/tpm2_unseal \
     tools/tpm2_dictionarylockout \
     tools/tpm2_createpolicy \
-    tools/tpm2_pcrextend
+    tools/tpm2_pcrextend \
+    tools/tpm2_convert_key
```

Comment:
I think we want is an auxiliary tools option that can be configured, but I am not 100% sold on it. IMHO, the tools should be things that use the TPM2.0 spec. Since this is just a helper, I would make an tools/aux/ folder for this and have configure --enable-aux-tools

Reply:
I don't know if this justifies the effort given that this new tool doesn't introduce any new dependencies or anything. Should a lot of these not stricly spec related tools accumulate it is something that could still be done then.

Reply:
We already tpm2_akparse. We can revisit this later.